cfgov/tccp/models.py

```diff
@@ -724,3 +724,17 @@ def annual_fee_estimated(self):
             fee += 52 * self.weekly_fee
 
         return fee
+
+    @property
+    def purchase_apr_data_incomplete(self):
```

Comment:
I love this so much as a way to flag incomplete purchase APR data! For a future PR, maybe we can think about extending that to intro APRs, too. There is only [a single card in the current dataset missing intro APR data](http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards/capital-one-national-association-savor-rewards/), though, so maybe not worth it (I didn't even try handling it gracefully since there's only one). Also, one thing I was thinking might make sense to refactor is to do the back-end calculations of all the APRs like we do for purchase APR, so maybe we could roll this into that work if we do it. Either way, this looks great for this PR.